tests/python/test_glm5_model.py

```
@@ -0,0 +1,612 @@
+
+No GPU or DashInfer C++ engine required — tests the Python logic only.
+We mock the C++ extension to test pure-Python model code.
+"""
+import sys
+import types
+import struct
+from unittest.mock import MagicMock
+from collections import defaultdict
+
+import torch
+import numpy as np
```

Flake8 (7.3.0)

[error] 'numpy as np' imported but unused

(F401)